modules/fbs/fbs.module

```diff
@@ -571,12 +571,23 @@ function fbs_form_profile2_phone_validate($element, $form_state) {
   $phone = $form_state['input']['profile_provider_fbs']['field_fbs_phone']['und'][0]['value'];
 
   if (!empty($phone)) {
-    // Preg match for danish phonenumbers
-    // Pass:
-    // (+45) 35 35 35 35 ||| +45 35 35 35 35 ||| 35 35 35 35 ||| 35353535
-    // Fail:
-    // (45)35353535 ||| 4535353535.
-    $re = '/^((\(?\+45\)?)?)(\s?\d{2}\s?\d{2}\s?\d{2}\s?\d{2})$/';
+    // Preg match for phonenumbers
+    // Pass (danish phonenumbers):
+    // (+45) 35 35 35 35 ||| +45 35 35 35 35 ||| 35 35 35 35 |||
+    // (+45)35353535 ||| +4535353535 ||| 35353535
+    // Pass (german phonenumbers):
+    // (+49) 089/12345678 ||| (+49) 089-12345678 ||| +49 089/12345678 ||| +49 089-12345678
+    // +49089/12345678 ||| +49089-12345678 ||| (+49)(089) 12345678
+    // Allows for countrycode with, and without parenthesis.
+    // Allows for spaces between numberpairs in number
+    // Allows for / and - in number (used in german phonenumbers).
+    // First matching group: ([\(]{0,1}[\+][0-9]{1,3}[ \.\-]?[\)]{0,1})
+    // Matches countrycode (with and without parenthesis)
```

Comment:
But this will match almost any country code - not just Denmark and Germany. Is that intentional?

Reply:
If the scope of the task has been updated then please make sure that the PR is updated accordingly.

It is quite confusing that multiple aspects refer to German phone numbers when the implementation is now intentionally not restricted to these two countries. This includes:
- Code comments
- Commit message
- PR title

The examples in the code comments, the commit message and event the PR title all refer to german phone numbers. It is quite co